server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java

```diff
@@ -29,17 +29,29 @@
 import org.apache.james.mailbox.quota.MaxQuotaManager;
 import org.apache.james.mailbox.quota.QuotaCount;
 import org.apache.james.mailbox.quota.QuotaSize;
+import org.apache.james.webadmin.jackson.QuotaModule;
+import org.apache.james.webadmin.service.DomainQuotaService;
+import org.apache.james.webadmin.utils.JsonTransformer;
 import org.eclipse.jetty.http.HttpStatus;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
+import com.google.common.collect.ImmutableSet;
 import com.jayway.restassured.RestAssured;
 import com.jayway.restassured.http.ContentType;
 import com.jayway.restassured.path.json.JsonPath;
 
-@ExtendWith(ScanningQuotaSearchExtension.class)
 class DomainQuotaRoutesTest {
+    @RegisterExtension
+    Extension extension = new ScanningQuotaSearchExtension(
+        testSystem -> new DomainQuotaRoutes(
+            testSystem.getDomainList(),
+            new DomainQuotaService(testSystem.getMaxQuotaManager()),
+            testSystem.getUsersRepository(),
+            new JsonTransformer(new QuotaModule()),
```

Comment:
I guess the last two lines could be declared outside the Extension to avoid QuotaModule duplicate instantiation